UnboundLib/UnboundLib.csproj.user.dist

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<PropertyGroup>
-		<RoundsFolder>C:\Program Files (x86)\Steam\steamapps\common\ROUNDS</RoundsFolder>
+		<RoundsFolder>A:\Program Files\SteamLibrary\steamapps\common\ROUNDS</RoundsFolder>
```

Comment:
Don't change the value here. Copy `UnboundLib.csproj.user.dist` to `UnboundLib.csproj.user` and change the folder there. The dist file is there just so that you can easily make a personal user config.

Reply:
Mistakenly pushed that change. Corrected

Reply:
The change to the `UnboundLib.csproj.user.dist` remains. You reverted the one to `UnboundLib.csproj`.